python/ray/data/_internal/execution/streaming_executor.py

```diff
@@ -352,14 +352,18 @@ def _consumer_idling(self) -> bool:
         return len(self._output_node.outqueue) == 0
 
     def _report_current_usage(self) -> None:
-        cur_usage = self._resource_manager.get_global_usage()
+        running_usage = self._resource_manager.get_global_running_usage()
+        pending_usage = self._resource_manager.get_global_pending_usage()
         limits = self._resource_manager.get_global_limits()
         resources_status = (
             "Running: "
-            f"{cur_usage.cpu:.4g}/{limits.cpu:.4g} CPU, "
-            f"{cur_usage.gpu:.4g}/{limits.gpu:.4g} GPU, "
-            f"{cur_usage.object_store_memory_str()}/"
-            f"{limits.object_store_memory_str()} object_store_memory"
+            f"{running_usage.cpu:.4g}/{limits.cpu:.4g} CPU, "
+            f"{running_usage.gpu:.4g}/{limits.gpu:.4g} GPU, "
+            f"{running_usage.object_store_memory_str()}/"
+            f"{limits.object_store_memory_str()} object_store_memory; "
+            "Pending: "
+            f"{pending_usage.cpu:.4g} CPU, "
+            f"{pending_usage.gpu:.4g} GPU"
```

Comment:
Nit, the `Running:` label above actually means this dataset is running. 
Maybe let's keep it as its original meaning. 
We can probably format this message as `Running. Resources: ... (pending: ...).`

Reply:
we may want to use symbols to shorten the message if it becomes too long.

Reply:
let's keep it as text for now, and we can have a separate PR to re-format and re-organize the progress bar output. i'll also want to include a docs update explaining the meaning of the symbols in that PR

Reply:
![Screenshot 2024-08-18 at 3 21 28 PM](https://github.com/user-attachments/assets/662dd144-fee8-4657-9ba3-5095e20b00e2)

Here is the adjusted progress bar. Note that a warning is raised as the progress bar has over 100 characters which is expected. Created an issue to use symbols to shorten the progress bar message.

Reply:
Tracking with #47190